assets/src/edit-story/utils/generatePatternStyles.js

```diff
@@ -94,7 +82,7 @@ function getGradientDescription({ type, rotation, center, size }) {
  *
  * @return {Array} List of serialized stops
  */
-function getStopList(stops, alpha, isAngular) {
+function getStopList(stops, alpha, isAngular = false) {
```

Comment:
`isAngular` is not needed now, as all both linear and gradient stops can be described using `%`. This parameter can be removed and logic below can be collapsed.

Reply:
Removed in 9a9fb4e